types/tengitsui/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false,
```

Comment:
You should also not disable this rule. As far as I can see, the lint error you were getting from this rule is that the type definition version didn't match an existing npm version of the original package. If you use an existing version, then you should be able to remove this line.